<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigApp</class>
 <widget class="QDialog" name="ConfigApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigApp</string>
  </property>
  <property name="styleSheet">
   <string>QDialog {
 
background-color:rgb(246,243,243);

}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>351</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>set message ring</string>
   </property>
   <widget class="QLineEdit" name="RingFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BT_ChangeRing">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>20</y>
      <width>37</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>change</string>
    </property>
   </widget>
   <widget class="QToolButton" name="BT_TestRing">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>20</y>
      <width>37</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QToolButton" name="BT_RingDefault">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>37</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>auto reply message</string>
   </property>
   <widget class="QTextEdit" name="MsgAutoRelpy">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>310</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_AutoReply">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>enable auto reply</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>227</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>message send mode</string>
   </property>
   <widget class="QRadioButton" name="RB_EnterSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter send message</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_CtrlEnterSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ctrl+Enter send message</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>general</string>
   </property>
   <widget class="QCheckBox" name="CB_MainTopHit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MainWind Top Hit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_AutoShowMsg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>auto show msg</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_RemberPwd">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>61</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>remember password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_MainStartHide">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>start hide windows</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_Mute">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>72</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_LongMsg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>long msg support</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_DisableNudge">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>80</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>disable nudge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_SetDefaultFont">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>default font</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_SetFont">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set font</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>get message(only on windows)</string>
   </property>
   <widget class="QLineEdit" name="hotKey">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>18</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_EnableHotKey">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>19</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>using hot key</string>
    </property>
   </widget>
   <widget class="QLabel" name="registeredHotKeyState">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>20</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
